themes/package.json

```diff
@@ -9,8 +9,8 @@
   },
   "dependencies": {
     "@noble/hashes": "^1.5.0",
-    "@patternfly-v5/patternfly": "npm:@patternfly/patternfly@^5.3.1",
-    "@patternfly/patternfly": "^5.4.1",
+    "@patternfly-v5/patternfly": "npm:@patternfly/patternfly@^5.4.1",
+    "@patternfly/patternfly": "^4.224.5",
```

Comment:
Looks like this version was accidentally upgraded by Dependabot. This is supposed to be the v4 version of PatternFly.